karma.conf.js

```diff
@@ -8,6 +8,7 @@ module.exports = function(config) {
     webpack: webpackConfig,
     frameworks: ['es5-shim', 'mocha', 'chai', 'sinon-chai', 'chai-jquery', 'jquery-2.1.0'],
     files: [
+      './node_modules/es6-shim/es6-shim.js',
```

Comment:
Not sure why PhantomJS is busted in this repo without the ES6 shim, but this approach is [recommended by others](https://github.com/monounity/karma-typescript/issues/224) for fixing this exact issue with Karma.

Reply:
I think phantom actually just doesn't support certain parts of es6...